pkg/cmd/server/server.go

```diff
@@ -149,6 +147,17 @@ func (serverOptions ServiceCatalogServerOptions) runServer() error {
 		return err
 	}
 
+	apiGroupInfo := genericapiserver.NewDefaultAPIGroupInfo(GroupName)
+	apiGroupInfo.GroupMeta.GroupVersion = schema.GroupVersion{
+		Group:   GroupName,
+		Version: watch.APIVersion,
```

Comment:
I feel like this should not come from the watch package, which TBH, I expect to go away.  I believe the pattern in k8s is that the scheme version lives in the version's `register.go` file.  Example:

https://github.com/kubernetes/kubernetes/blob/master/pkg/master/master.go#L388
https://github.com/kubernetes/kubernetes/blob/master/pkg/api/v1/register.go#L30